server/src/main/java/org/apache/cassandra/sidecar/routes/RingHandler.java

```diff
@@ -95,15 +107,15 @@ protected void processFailure(Throwable cause,
             return;
         }
 
-        super.processFailure(cause, context, host, remoteAddress, keyspace);
+        super.processFailure(cause, context, host, remoteAddress, request);
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
-    protected Name extractParamsOrThrow(RoutingContext context)
+    protected Void extractParamsOrThrow(RoutingContext context)
     {
-        return keyspace(context, false);
+        return null;
```

Comment:
Why are we returning null?

Reply:
For Ring call across keyspaces, we do not take in keyspace. Hence using the default implementation of taking in void request.